js/src/components/social/FacebookWrapper.js

```diff
@@ -23,7 +23,10 @@ const FacebookWrapper = ( props ) => {
 			{
 				props.isPremium
 					? <Slot
-						name="YoastFacebookPremium"
+						name={
+							"YoastFacebookPremium" +
+							`${ props.location ? props.location.charAt( 0 ).toUpperCase() + props.location.slice( 1 ) : "" }`
```

Comment:
Since your defaultProp for `location` is `""`, you do not need to do the `props.location` check.

`"".charAt( 0 ).toUpperCase() + "".slice( 1 )` = `""` (yay, JavaScript!) and `"YoastFacebookPremium"` + `""` = `"YoastFacebookPremium"` :smile: